### DISCUSSION TITLE
Adding a Client using MAC doesn't match to device while using AGH DHCP server

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3220] [Q&A]
AdGuard Home Version: v0.106.3
OS: Ubuntu Server 20.04.4
Install Method: Snap

I can add a Client using the MAC address of the device, but it doesn't seem to match against AGH's data. The only way I can get it to match properly is by using the IP address as the client identifier. DHCP server is disabled at the router, and no other DHCP servers running except for AGH's

![image](https://user-images.githubusercontent.com/13022116/120376842-59d1f580-c2ea-11eb-8366-e8e47423bcc2.png)

Looking at a specific device (my laptop), I confirmed via IPCONFIG I'm using the AGH DHCP server, and the MAC reported here matches the MAC I used for the client identifier

![image](https://user-images.githubusercontent.com/13022116/120377469-28a5f500-c2eb-11eb-93ca-cb5c9239f480.png)

Here is the same device listed in the Clients (runtime) section of AGH: 
![image](https://user-images.githubusercontent.com/13022116/120377827-a2d67980-c2eb-11eb-881c-418220ff4482.png)

Am I missing something?

**Comment by ameshkov** [comment]
What Mac address do you see in the active leases list on DHCP settings page?

**Reply by ghost** [reply]
The same one I see using IPConfig

**Reply by ainar-g** [reply]
@ameshkov, the stats are currently only collected for Client-IDs and IP addresses.  And it seems that that has been the case, well, always.  That shouldn't be a problem once we choose a better storage, including for stats.

**Reply by ameshkov** [reply]
Yep, indeed. @NortonBW so it's a known issue then. Note, that Client settings are applied just okay, the problem is only with the requests count on that screen.

**Reply by ghost** [reply]
Thank you both for your help. I will keep an eye out for future updates that address this.
